tests/quality/conftest.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+import pytest
+
+import dspy
+from tests.conftest import clear_settings
+from tests.quality.utils import parse_quality_conf_yaml
+
+# Standard list of models that should be used for periodic DSPy quality testing
+MODEL_LIST = [
+    "gpt-4o",
+    "gpt-4o-mini",
+    "gpt-4-turbo",
+    "gpt-o1-preview",
+    "gpt-o1-mini",
+    "claude-3.5-sonnet",
+    "claude-3.5-haiku",
+    "gemini-1.5-pro",
+    "gemini-1.5-flash",
+    "llama-3.1-405b-instruct",
+    "llama-3.1-70b-instruct",
+    "llama-3.1-8b-instruct",
+    "llama-3.2-3b-instruct",
+]
```

Comment:
It's probably useful for us to define a canonical list of models that DSPy tests against. We can extend / update this list over time. For each model in this list, users can specify corresponding LiteLLM configurations in `quality_conf.yaml`, as described in the README